### DISCUSSION TITLE
下面的小箭头不行了？

### DISCUSSION THREAD

**Discussion by SednaMi** [share121/inter-knot#1765] [其他]
![image](https://github.com/user-attachments/assets/5cb57cb9-f838-42ff-9aff-f34355dbfeff)
从Discussion里面发的

**Comment by user1-cloud** [comment]
更新：已经修复好了

**Reply by user1-cloud** [reply]
@VacuolePaoo

**Reply by SednaMi** [reply]
希望尽快修复吧，每次发帖都要跑到discussions里面
